static/third/marked/lib/marked.js

```diff
@@ -647,8 +645,10 @@ InlineLexer.prototype.output = function(src) {
 
     // linkifier (Zulip)
     var self = this;
-    this.rules.linkifiers.forEach(function (linkifier) {
-      var ret = self.inlineReplacement(linkifier, src, function(regex, groups, match) {
+
+    const regexes = this.options.get_linkifier_regexes ? this.options.get_linkifier_regexes() : [];
```

Comment:
I think I'd prefer to not require `get_linkifiers_regexes` rather than having it fallback to `[]` here, in that it would protect us from a typo in the name causing the linkifiers configuration to be accidentally lost.

Reply:
I am a little confused by your wording here.  We have test coverage that prevents us from typos causing any issues, so your concerns are mostly hypothetical for the main codepath (rendering messages).  But if `get_linkifier_regexes` is not set for some other codepath (such as rendering zcommand responses), then should I just make the code fail hard?

Reply:
Yeah, my thinking was it'd be better to have this just fail hard (so the caller would pass `[]` if its intent was to not enable linkifiers).

Reply:
It’s a mostly moot point, but I would really prefer to fail soft here.  Linkifiers are not mission critical.

Reply:
To me, it's not a question of whether it's mission-critical; it's a question of whether it'd be a bug for `this.options.get_linkifier_regexes` to not be specified. I think the answer is that it would be in all non-test usage of this module, and so we might as well have an implicit assertion that it be.